pom.xml

```diff
@@ -199,20 +199,20 @@
                                 <ts.global.s2i.quarkus.native.builder.image>${ts.global.s2i.quarkus.native.builder.image}</ts.global.s2i.quarkus.native.builder.image>
                                 <!-- Services used in test suite -->
                                 <keycloak.image>quay.io/keycloak/keycloak:15.0.1</keycloak.image>
-                                <rhsso.73.image>registry.access.redhat.com/redhat-sso-7/sso73-openshift</rhsso.73.image>
-                                <postgresql.13.image>quay.io/bitnami/postgresql:13.4.0</postgresql.13.image>
-                                <mysql.57.image>quay.io/bitnami/mysql:5.7.32</mysql.57.image>
+                                <rhsso.image>registry.access.redhat.com/rh-sso-7/sso75-openshift-rhel8</rhsso.image>
+                                <postgresql.13.image>quay.io/bitnami/postgresql:13.5</postgresql.13.image>
+                                <mysql.57.image>quay.io/bitnami/mysql:5.7</mysql.57.image>
                                 <mysql.80.image>registry.access.redhat.com/rhscl/mysql-80-rhel7</mysql.80.image>
```

Comment:
Use `rhscl` as in case of mariadb

Reply:
And let's see how CI ends, maybe we will need secrets to pull from RH registry

Reply:
I'm gonna resolve this as it is irrelevant for now